Blockzilla/AppDelegate.swift

```diff
@@ -183,7 +183,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
             if application.applicationState == .active {
                 // If we are active then we can ask the BVC to open the new tab right away.
                 // Otherwise, we remember the URL and we open it in applicationDidBecomeActive.
-                browserViewController.openOverylay(text: text)
+                if let fixedUrl = URIFixup.getURL(entry: text) {
```

Comment:
Not sure if relevant but I discovered last week that `URIFixup.getURL` is very broken .. https://github.com/mozilla-mobile/focus-ios/issues/1927

Reply:
Ah, it would be nice to have a method that does what this method advertises to be doing then. 
I've tested it with URLs such as "https://www.mozilla.org/" and "www.mozilla.org" and seemed to behave as expected. 

I've noticed #1927 is assigned to yourself, I wouldn't mind writing some test cases for it if it helps.